proc/proc_windows.go

```diff
@@ -127,6 +127,20 @@ func Launch(cmd []string) (*Process, error) {
 		return nil, ProcessExitedError{Pid: dbp.Pid, Status: exitCode}
 	}
 
+	for _, thread := range dbp.Threads {
+		_, err := _SuspendThread(thread.os.hThread)
```

Comment:
We need to wrap this in `execPtraceFunc` as well, correct? I assume it has the same origin thread restrictions.